res/code/bml.php

```diff
@@ -107,28 +106,28 @@ function getutf8($s) {
   $build = $ver_array[2];
   
   // Read the KeymanWeb code from s.keyman.com/
-  $KeymanWebRoot = "{$KeymanCloudRootPath}kmw\\engine\\{$ver}";
+  $KeymanWebRoot = "https://s.keyman.com/kmw/engine/{$ver}";
 
   // We always load latest stable, which at 10.0 and later, uses `keyman` as global var
   $kmwbase = "keyman";
  
   if($debug) {
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\kmwstring.js"));
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\kmwbase.js"));
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\keymanweb.js"));
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\kmwosk.js"));
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\kmwnative.js"));
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\kmwcallback.js"));
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\kmwkeymaps.js"));
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\kmwlayout.js"));
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\kmwinit.js"));
-    echo getutf8(file_get_contents("{$KeymanWebRoot}\\src\\kmwuitoggle.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/kmwstring.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/kmwbase.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/keymanweb.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/kmwosk.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/kmwnative.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/kmwcallback.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/kmwkeymaps.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/kmwlayout.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/kmwinit.js"));
+    echo getutf8(file_get_contents("{$KeymanWebRoot}/src/kmwuitoggle.js"));
```

Comment:
Won't these paths only work for legacy KMW (pre Typescript refactoring)?

Reply:
Great catch. @jahorton what are the correct paths for inclusion now?

Reply:
With TS, everything's compiled into a single file - you'll only want `/keymanweb.js` and `kmwuitoggle.js`.

The trick is that for `if($debug)`, you'll want the _release/unminified/web_ version, rather than the plain _release/web` version.  I'm not sure what the corresponding file organization is on the servers, though.

Reply:
oh duh my bad. Okay, I've disabled debug mode for bookmarklet for now. It was only used internally by us anyway.